.github/workflows/stage_pr.yaml

```diff
@@ -2,38 +2,31 @@ name: "Deploy Pull Request"
 
 on: 
   pull_request:
```

Comment:
Looking back at this and Avent's recent PR I remember changing this line (at least locally) to [`pull_request_target`](https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target) which should allow outside contributors to use the action as if they made a pull request from within the repo. However this comes with the security concern of giving secret access to anyone that is able to run an action through pull requests. While the AWS role that is currently stored in the secrets should have pretty limited account access it still is a concern. Wondering if the cons are worth the benefit.